tests/conftest.py

```diff
@@ -74,7 +79,8 @@ def mock_account(api_url, account_id):
             "name": "Ben Bitdiddle",
             "object": "account",
             "provider": "gmail",
-            "organization_unit": "label"
+            "organization_unit": "label",
+            "billing_state": "trial",
```

Comment:
Can you change this so the default is `paid` please? This notion of trial will be deprecated soon so I'd like to avoid using it where possible (we're moving towards `paid`, `cancelled`, and `deleted` only). `paid` is the default state for all accounts, while downgrading will move them to `cancelled`. Sorry you didn't know this ahead of time :/